.Gemfile-rest-client-1.8.rb

```diff
@@ -0,0 +1,11 @@
+# Allow travis to test rest-client 1.x.
```

Comment:
@cben Can you create a pr to bump rest-client ? we should discuss that desperately
(I think we should do it, rest client < 2.0 has not seen updates in over two years)